sdk/src/main/java/com/horizen/cryptolibprovider/ThresholdSignatureCircuitImplZendoo.java

```diff
@@ -91,4 +87,15 @@ public int sysDataConstantLength() {
 
         return sysDataConstantBytes;
     }
+
+    @Override
+    public int sysDataConstantLength() {
+        return PoseidonHash.HASH_LENGTH;
+    }
+
+    @Override
+    public int proofSizeLength() {
+        return 771; //@TODO take it from JNI side
+    }
```

Comment:
Add also the method for VkCertKey length form sidechain creation output

Reply:
done